src/modules/items/server/events/onDatabaseReady.lua

```diff
@@ -0,0 +1,13 @@
+NCS:handleEvent("databaseReady", function()
+    API_Database:query("SELECT * FROM ncs_items", {}, function(rows)
+        if (#rows <= 0) then
+            return (NCS:die("You must have at least one item in your database"))
+        end
+        for _, row in pairs(rows) do
+            ---@type NCSItems
+            local item = NCSItems(row.name, row.label, row.weight)
+            MOD_Items:set(row.name, row.label, row.weight)
+            NCS:triggerClientEvent("receiveItems", -1, item:minify())
+        end
+    end)
+end)
```

Comment:
we could store the items in the scripts.
If it were a shared type, you would not have to trigger items.
See: [link](https://github.com/NextCitizens/ncs_core/tree/main/src/constant)